content/200-orm/100-prisma-schema/50-introspection.mdx

```diff
@@ -29,8 +29,7 @@ Here's an overview of its main functions on SQL databases:
 
 On MongoDB, the main functions are the following:
 
-- Map _collections_ in the database to [Prisma models](/orm/prisma-schema/data-model/models#defining-models)
-- Map _documents_ in the collections to the [fields](/orm/prisma-schema/data-model/models#defining-fields) of Prisma models by _sampling them_
+- Map _collections_ in the database to Prisma models. Because a _collection_ in MongoDB doesn't have a predefined structure, Prisma ORM _samples_ the _documents_ in the collection and derives the model structure accordingly. If _embedded types_ are detected in a collection, these will be mapped to composite types in the Prisma schema.
```

Comment:
```suggestion
- Map _collections_ in the database to Prisma models. Because a _collection_ in MongoDB doesn't have a predefined structure, Prisma ORM _samples_ the _documents_ in the collection and derives the model structure accordingly (i.e. it maps the [fields](/orm/prisma-schema/data-model/models#defining-fields) or the _document_ to the fields of the model). If _embedded types_ are detected in a collection, these will be mapped to composite types in the Prisma schema.
```